run_dir/static/js/workset_samples.js

```diff
@@ -177,8 +187,14 @@ function updatePlate(level){
     if(wsdata.hasOwnProperty("projects")){
         $.each(wsdata.projects, function(project_id, project_data){
             $.each(project_data.samples, function(sample_id, sample_data){
-                updateSample(sample_id, sample_data, level);
+                updateSample(sample_id, sample_data, level, is_control_project(project_data.name));
             });
         });
     }
 }
+
+function is_control_project(project){
+    if(project.toLowerCase().startsWith('n.egative')|| project.toLowerCase().startsWith('p.ositive'))
```

Comment:
I was hoping we could use the `Type: Control` to differentiate them, but maybe that's a lot more work? I would be OK with this as well

Reply:
We decided to stick to the name since we anyhow need to determine if it's a positive or negative one, not just a control.